recipes/nmslib/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "nmslib" %}
+{% set version = "2.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6a72b132b4b7fe7d697767005c69bc66c8fbc83dbba2d07109c0da29c0f00353
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }} 
+  script:
```

Comment:
Script needs to be up in the top-level `build` section, not under requirements (i.e at the same level as `number` above.

Reply:
Oops fixed. I still have the `not compiled to use: SSE4.1 SSE4.2 AVX AVX2` warning I mentioned above though. This warning seems to be os dependent too (mac osx: SSE4.1 SSE4.2 AVX; windows : AVX AVX2).

Reply:
I'm not entirely sure what's going on here. 

@conda-forge/help-python-c do you have any suggestions?

Reply:
I looked it up more maybe this is expected behavior since conda-forge should provide binaries that are compatible with most CPUs, including those that do not support those instructions. I am ok with leaving these as-is unless there is a better solution then.